OmiseSDK/Client.swift

```diff
@@ -160,7 +160,8 @@ extension Client {
     }
     
     private func buildCapabilityAPIURLRequest() -> URLRequest {
-        let urlRequest = NSMutableURLRequest(url: URL(string: "https://api.omise.co/capability")!)
+        let configuration = Configuration()
+        let urlRequest = NSMutableURLRequest(url: configuration.environment.capabilityURL)
```

Comment:
Maybe it would be a good opportunity to replace the old `NSMutableURLRequest` with `URLRequest`? ;)